From 38db0ecdd2af7712b8c8a83f619076645f670208 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Sun, 25 Jun 2017 07:40:36 -0500
Subject: [PATCH] "0001-validus-build-default-ext4-userimages
 _______________________________

*Previous From b0b7e99b42101272bec14121a24d674e376d2bf6 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Sat, 24 Jun 2017 13:59:35 -0500
*Previous Subject: [PATCH] validus: build OTATOOLS for non-generic devices
*
*-mkuserimg.sh is not in $PATH and therefore fails when
*executed from releasetools. I have not tested this yet.
*What I did was copy mkuserimg.sh from los14
*to out/host/linux-x64/bin to get it to work. So, test this
*the next chance I get.
*
*-Default to ext4 fs format when creating images."
---
 tools/releasetools/build_image.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/releasetools/build_image.py b/tools/releasetools/build_image.py
index 80c1db2..af3b818 100755
--- a/tools/releasetools/build_image.py
+++ b/tools/releasetools/build_image.py
@@ -374,7 +374,7 @@ def BuildImage(in_dir, prop_dict, out_file, target_out=None):
       fs_config = merged_fs_config
 
   build_command = []
-  fs_type = prop_dict.get("fs_type", "")
+  fs_type = prop_dict.get("fs_type", "ext4")
   run_fsck = False
 
   fs_spans_partition = True
-- 
1.9.1

